Makefile

```
@@ -0,0 +1,47 @@
+.PHONY: help lint lint-fix serve build install-deps clean
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)